packages/stockflux-launcher/src/popups/ConfirmationWindow.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PopupWindow from './PopupWindow';
+import { FaCheck, FaTimes } from 'react-icons/fa';
+
+const ConfirmationWindow = ({ message, onConfirm, onReject, children }) => {
+  const modalOptions = [
+    {
+      name: 'Cancel',
+      icon: <FaTimes></FaTimes>,
+      className: 'modal-cancel',
+      action: onReject
+    },
+    {
+      name: 'Confirm',
+      action: onConfirm,
+      icon: <FaCheck></FaCheck>,
+      className: 'modal-confirm'
```

Comment:
Nitpick, but can the variables be kept in the same order for readability purposes?

Reply:
The order in which the items appear in the list are the same as they appear in the window. I think we want the OK button to be the bottom left icon with the cancel to the right of it, it might be a question for UX on order of buttons.